clients/da-vinci-client/src/test/java/com/linkedin/davinci/config/DaVinciConfigTest.java

```diff
@@ -43,7 +42,7 @@ public void testRecordTransformerEnabled() {
 
   @Test
   public void testGetAndSetRecordTransformer() {
-    DaVinciRecordTransformer transformer = new TestRecordTransformer();
+    DaVinciRecordTransformer transformer = new TestRecordTransformer(0);
```

Comment:
We'll need to call new on this in the IngestionTask in order to supply the version, So we probably need a factory of some kind passed to the davinci client.